client/ayon_core/hosts/houdini/plugins/create/create_hda.py

```diff
@@ -84,5 +110,22 @@ def create(self, product_name, instance_data, pre_create_data):
 
     def get_network_categories(self):
         return [
-            hou.objNodeTypeCategory()
+            category for name, category in hou.nodeTypeCategories().items()
+            if name in {
+                "Chop", "Cop2", "Dop", "Driver", "Lop",
+                "Object", "Shop", "Sop", "Top", "Vop"
+            }
```

Comment:
Is there a reason we prefer this over explicitly providing it the type categories directly as in ` hou.chopNodeTypeCategory(), hou.cop2NodeTypeCategory(), ...`?

Reply:
> Is there a reason we prefer this over explicitly providing it the type categories directly as in ` hou.chopNodeTypeCategory(), hou.cop2NodeTypeCategory(), ...`?

Maybe because we have talked about the hda node can offer more than one outputs at SOP level but only one output at Obj level? But `hou.objNodeTypeCategory()` is already returning to all the node type categories so we dont really need to change this.

Reply:
> Is there a reason we prefer this over explicitly providing it the type categories directly

I found it easier to read and maintain because category names are easier to remember.

Reply:
Are explicit category type more favored ?

Reply:
Fair! 
Thank you for the tip. 
implemented in https://github.com/ynput/ayon-core/pull/456/commits/79139ce0dd4b31c32c98d26403a2f78c6e345f14